### DISCUSSION TITLE
Lodestar Standup 2024-17 (April 30, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6716] [Standups]
### Meeting Info
[Apr 30, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/apr-30-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - Miga Labs Hardware Analysis
  - BLST, Yamux, N-historical states
  - Docs: Versioning + Content reorganization 
    - #6669
    - #6550
- Team Updates
  - Head of Protocol position
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by twoeths** [comment]
I propose we create check list for the interop to track different items of each EIP, this is to make sure we well prepare for the event

**Comment by twoeths** [comment]
an update about n-historical state:
- run well for 7 days in `feat4` with 2 configurations:
  - default (2 epoch checkpoint states, 32 block states), nodes are suppose not to reload states frequently with this config. In fact I haven't seen any so far
  - maintain mostly 1 block (head) state: nodes have to reload states multiple times, it's been great so far
- my proposed plan is to test this for the whole `v1.18.0` period to make sure it works well first before going further

Some TODOs:
- finalized state is normally persisted instead of in-memory so need to get through all apis to deal with that
- Rss is spiked to 30GB on `1k` node right after the start then get back to normal after ~9h, it does not happen on other nodes

updated the status in https://github.com/ChainSafe/lodestar/issues/5968

also I was told there's gonna be a break out session in the interop event regarding a long unfinality test network, it would be great to test this work with that network

**Comment by philknows** [comment]
Closing for #6728
